java/core/src/test/java/com/microsoft/bond/testmodel/MyEnum.java

```diff
@@ -78,6 +78,30 @@ public static MyEnum get(int value) {
         }
     }
 
+    public static MyEnum parse(String str) {
```

Comment:
Would it make sense for there to be two methods: `tryParse` that returns `null` when `str` doesn't match anything and `parse` that throws (implemented by calling `tryParse` and throwing if the return is `null`)?